content/working_with/console/widgets/serviceButton.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: bt_wiki
+title: Service button
+category: Widgets
+draft: false
+---
+
+Displays a button allowing to upload a blueprint from the configured blueprint catalog/marketplace. 
```

Comment:
Looks like we are using terms 'catalog' and 'marketplace' interchangeably. Would be good to decide to use only one of them consistently.

Reply:
I know, it's hard to say which one would be the bets here. Blueprint Catalog widget and changes in Blueprints widget should be documented first and then I could refer to it probably.

I'll go with 'marketplace' for now.